protected/tests/functional/SiteTest.php

```diff
@@ -46,6 +46,26 @@ public function testItShouldShowLatestNews()
 
     }
 
```

Comment:
I think it would be good to add a descriptive comment block for this function like the `testItShouldShowLatestNews()` function above.